lms/static/js/spec/student_account/account_settings_fields_spec.js

```diff
@@ -0,0 +1,337 @@
+
+                // Verify server side issue/failure message
+                textFieldView.$(textFieldSelector).val(data.name).change();
+                fieldValueChangeEventHandler(this, data, 500);
+                expect(textFieldView.$(messageSelector).text().trim()).toBe('An error occurred, please try again.');
+
+                // Verify wrong input message
+                // Empty the text field which is not a valid value, so we should get an error message
+                textFieldView.errorMessage = '<i class="fa fa-exclamation-triangle"></i> Invalid Full Name value.';
+                textFieldView.$(textFieldSelector).val('').change();
+                fieldValueChangeEventHandler(this, {'name': ''}, 400);
+                expect(textFieldView.$(messageSelector).text().trim()).toBe('Invalid Full Name value.');
+            });
+
+            it("can correctly render the dropdown field help messages", function() {
+                var fieldValueSelector = '.account-settings-field-value > select';
+                var messageSelector = '.account-settings-field-message';
+                var data = {'language': 'si'};
+                var fieldData = createFieldData(AccountSettingsFieldViews.DropdownFieldView, {
+                    title: 'Language',
+                    valueAttribute: 'language',
+                    required: true,
+                    options: MOCK_LANGUAGE,
+                    helpMessage: 'edX Languge'
+                });
+                var dropdownFieldView = new AccountSettingsFieldViews.DropdownFieldView(fieldData).render();
+                requests = AjaxHelpers.requests(this);
+
+                // Verify default message
+                expect(dropdownFieldView.$(messageSelector).text().trim()).toBe(fieldData.helpMessage);
+
+                // Verify change in-progress message
+                dropdownFieldView.$(fieldValueSelector).trigger('change');
+                expect(dropdownFieldView.$(messageSelector).text().trim()).toBe('Saving...');
+
+                // Verify change completion message
+                dropdownFieldView.$(fieldValueSelector).trigger('change');
+                fieldValueChangeEventHandler(this, data);
+                expect(dropdownFieldView.$(messageSelector).text().trim()).toBe('Successfully changed.');
+
+                // Verify server side issue/failure message
+                dropdownFieldView.$(fieldValueSelector).trigger('change');
+                fieldValueChangeEventHandler(this, data, 500);
+                expect(dropdownFieldView.$(messageSelector).text().trim()).toBe('An error occurred, please try again.');
+
+                // Verify wrong input message
+                // Empty the text field which is not a valid value, so we should get an error message
+                dropdownFieldView.errorMessage = '<i class="fa fa-exclamation-triangle"></i> Invalid language value.';
```

Comment:
add an ```aria-hidden=true``` attribute on the icon font and a screen reader accessible label i.e. ```<span class="sr">Error:</span>``` as  a sibling to the icon font (between the icon and the actual error msg)